### ISSUE TITLE
Support GeoPackage format

### ISSUE THREAD

**Issue by fzadrazil** [LayerManager/layman#612]
Some users requested a geopackage data source for publishing layers.

**Comment by jirik** [comment]
Another option is to change Layman to accept also GeoPackage with multiple layers and to publish all such layers. It would require some system changes in Layman, but it sounds more systematic to me than to extract GeoPackage layers at client and send them one by one to Layman.

**Comment by jmacura** [comment]
GPKG implementation guide might be helpful: http://www.geopackage.org/guide/implementation_guide.html
